### PR TITLE
Run2-TB26 Add possible scenario for 2017 HGCAL TB

### PR DESCRIPTION
**Pull request by bsunanda**
The default scenario is updated and 3 other scenario's added for 2017 TB which are foreseen as of now.

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: SimG4CMS/HGCalTestBeam/test/waferGeom.C [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#include <algorithm>
```
Comment (ianna) [inline SimG4CMS/HGCalTestBeam/test/waferGeom.C]: @bsunanda - why is the file extension .C and not .cc?

**Review by ianna — CHANGES_REQUESTED** [review]

File: SimG4CMS/HGCalTestBeam/test/waferGeom.C (hunk 1 above)
Comment (bsunanda) [inline SimG4CMS/HGCalTestBeam/test/waferGeom.C]: This is used for analyzing the MC data and used within ROOT. I made these functions and the analyzers of TB data use them and asked me to put it in the repository

Comment (ianna) [inline SimG4CMS/HGCalTestBeam/test/waferGeom.C]: @bsunanda - I'd suggest to add a macros directory and put it there. For example: http://cmslxr.fnal.gov/source/Fireworks/Geometry/macros/

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ianna** [comment]
+1

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ianna** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by bsunanda** [comment]
@kpedro88 Please approve

**Comment by kpedro88** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**